composer.json

```diff
@@ -3,17 +3,31 @@
 	"description": "PrestaShop module ps_mbo",
 	"homepage": "https://github.com/PrestaShop/ps_mbo",
 	"license": "AFL-3.0",
+	"type": "prestashop-module",
 	"authors": [
 		{
 			"name": "PrestaShop SA",
 			"email": "contact@prestashop.com"
 		}
 	],
+	"config": {
```

Comment:
take time to move to spaces instead of tabs :)

Reply:
Done, thanks